### PR TITLE
you can shit wherever you want. handles issue #5

### LINKED ISSUE #5: Shit only in an available bathroom

### PR DESCRIPTION
**Pull request by n0pan**
image

### CODE REVIEW

File: teleport.js [hunk 1]
```diff
@@ -196,18 +196,67 @@ function setDesk() {
...
-    console.error("No toilets found on the current map.");
+    console.error("Map doesn't have a bathroom.");
+  }
+
+  // get non-occupied toilets
+  const currentPlayersInMap = getPlayers().filter(
+    p => p.map === selectedMap.id
+  );
+  toilets.forEach((t, toiletSeatIndex) => {
+    const playerAtSeat = currentPlayersInMap.filter(
```
Comment (michmich112) [inline teleport.js]: rename to `playerTakingAShit` as an improved variable name

**Review by michmich112 — COMMENTED** [review]
great that you added the handling of other players shitting
i offered an alternate variable name

**Review by michmich112 — APPROVED** [review]

File: teleport.js (hunk 1 above)
Comment (n0pan) [reply teleport.js]: ah yes my bad

**Review by michmich112 — APPROVED** [review]
image

**Pull request merged**